core/templates/dev/head/footer.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <style>
+    .footer{
+    position: relative; 
+    background-color: #094142;
+    width: 100%;
+    text-align: left;
+    padding: 55px 50px 50px 200px;
+    }
+
+    .footer .footer-column{
+    display: inline-block;
+    vertical-align: top;
+    width: 20%;
+    line-height: 20px;
+    font-size: 13px;
+    }
+
+    .footer .footer-column a{
+    color: white;
+    text-decoration: none; 
+    }
+
+    .footer .footer-column span{
+    display: block;
+    color: white;
+    font-size: 14px;
+    }
+  </style>
+
+  <footer class="footer">
+    <div class="footer-column">
+      <span>ABOUT OPPIA</span>
+      <a href="/about">About</a><br>

```

Comment:
Prefer not to use <br>. This is really a list, so can you use ul/li and add CSS to remove the bullet points?